content/en/boilerplates/gateway-api-install-crds.md

```diff
@@ -5,5 +5,5 @@ installed before using the Gateway API:
 
 {{< text syntax=bash snip_id=install_crds >}}
 $ kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
-  { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref={{< k8s_gateway_api_version >}}" | kubectl apply -f -; }
```

Comment:
I assumed the kustomize magic was to stop us having to update the boilerplate when a new version comes out.  Though in typing that I realised we do have to update it in `k8s_gateway_api_version` and so that doesn't make sense.

Should you `s/v1.1.0/{{< k8s_gateway_api_version >}} ` at least?

Reply:
I've now done a bunch of spelunking through the Gateway API site and have no idea why they actually recommended using kustomize (and if they changed this recommendation).

It looks like we can nuke anywhere it was used from now on and only use standard-install or experimental-install.  Feel free not to make those changes in this PR.

Reply:
Yeah that tracks with what I saw - I know for a fact it's not used (or recommended) for their standard CRD install anyway, and I've confirmed waypoints work and deploy just fine without it.

Reply:
> Should you s/v1.1.0/{{< k8s_gateway_api_version >}} at least?

oh I missed we do this, and "wait why don't we automunge the version" crossed my teeny brain at least once while looking at this.